src/spring-mongo/src/main/java/net/conology/spring/restjsonpath/mongo/ir/MongoAlternativesSelector.java

```diff
@@ -0,0 +1,7 @@
+package net.conology.spring.restjsonpath.mongo.ir;
+
+public sealed interface MongoAlternativesSelector
```

Comment:
BooleanSelector/LogicalSelector might be a better name. Okay for now. Let's look at it, once we introduce negation